ddl/util/util.go

```diff
@@ -128,8 +129,12 @@ func RemoveMultiFromGCDeleteRange(ctx sessionctx.Context, jobID int64, elementID
 		idList += "%?"
 		paramIDs = append(paramIDs, elementID)
 	}
-	completeDeleteMultiRangesSQL := completeDeleteMultiRangesSQLPrefix + idList + completeDeleteMultiRangesSQLSuffix
-	_, err := ctx.(sqlexec.SQLExecutor).ExecuteInternal(context.TODO(), completeDeleteMultiRangesSQL, paramIDs...)
+	var buf strings.Builder
+	_, err := fmt.Fprintf(&buf, completeDeleteMultiRangesSQL, idList)
```

Comment:
Just let you know, we have `fmt.Fprintf` in our errcheck_ignore lists. And I agreed with that. `fmt.Fprintf` can not throw error on valid input, unless OOM. So you do not need to check that error.

BTW, You could use `Fprintf` to concat `idList` & `nameList` too.